index.js

```diff
@@ -62,7 +62,14 @@ function foregroundChild (...fgArgs) {
     // Allow the callback to inspect the child’s exit code and/or modify it.
     process.exitCode = signal ? 128 + signal : code
 
-    cb(() => {
+    let done = false;
+    const doneCB = () => {
+      /* istanbul ignore if: paranoid check */
```

Comment:
Shouldn't this be testable by returning a promise and then calling done after it resolves? (Ie, I don't know it's actually paranoid.)

Reply:
I don't think this can be tested using promises but I added a test for calling `done` twice.  This required stubbing `process.exit` to break the fact that `done` normally does not return.